app/src/main/java/com/android/sample/model/recipe/RecipesViewModel.kt

```diff
@@ -24,8 +24,7 @@ class RecipesViewModel(private val repository: RecipeRepository) : ViewModel() {
     get() = _loading
 
   init {
-    // Fetch a default number of random recipes when ViewModel is created
-    fetchRandomRecipes(5) // Fetch 5 random recipes as an example
+    // Removed the fetchRandomRecipes() call from the init block for testing purposes
   }
 
   /**
```

Comment:
Please ensure that fetchRandomRecipes properly handles cases based on the value of numberOfRecipes (e.g., when it's less than 0, equal to 0, or greater than 0). Consider whether this logic should be implemented in the RecipeRepository.

Reply:
The fetchRandomRecipes function includes now a check to ensure numberOfRecipes is at least 1.